widget/view.html

```diff
@@ -3,62 +3,77 @@
+                        <div class="connector-card-secondary-header-text"><span>{{item.configuration.length}}
+                            {{(item.configuration.length === 1)?'Configuration':'Configurations'}} Monitored</span></div>
+                      </div>
+                    </div>
+                    <div class="margin-left-auto;">
+                      <div class=" padding-right-sm">
+                        <span class="connector-button fa fa-lg"
+                          data-ng-class="{'fa-question-circle': item.connectorHealthStatus === 'unknown', 'fa-check-circle': item.connectorHealthStatus === 'available', 'fa-exclamation-circle': item.connectorHealthStatus === 'disconnected'}"></span>
+                        <span
+                          class="connector-health-status-message {{item.connectorHealthStatus}}">{{item.connectorHealthStatusMessage}}</span>
+                        <span class="fa config-collapse-btn"
+                          data-ng-class="{'fa-caret-down': collapse, 'fa-caret-up': !collapse}"></span>
+                      </div>
+                    </div>
                   </div>
                   <div class="connector-config-wrapper">
                     <div data-ng-hide="collapse" class="col-md-12 connector-config">
                       <div class="row margin-top-md">
                         <div class="col-md-6">
-                          <span class="pull-left muted font-10px">Configuration Name</span>
+                          <span class=" muted font-10px">Configuration Name</span>
                         </div>
                         <div class="col-md-6">
-                          <div class="pull-right">
+                          <div class="position-absolute right-10">
                             <span class="config-status-wrapper muted font-10px"> Health Check Status</span>
                           </div>
                         </div>
                       </div>
-                      <div class="config-health-status row" data-ng-repeat="items in item.configuration track by $index">
+                      <div class="config-health-status row"
+                        data-ng-repeat="items in item.configuration track by $index">
                         <div class="col-md-6 padding-left-0 padding-right-sm">
-                          <span class="pull-left connector-config-header-text">{{items.name}} <span data-ng-if="items.agentName"> - {{items.agentName}}</span></span>
+                          <span class="connector-config-header-text">{{items.name}} <span data-ng-if="items.agentName">
+                              - {{items.agentName}}</span></span>
                         </div>
                         <div class="col-md-6 padding-right-0 padding-left-sm">
-                          <div class="pull-right">
+                          <div class="position-absolute config-health-message-position">
```

Comment:
position absolute not required

Reply:
If position-absolute not used Right doesnt work, to stick it to right I've used position absolute

Reply:
position-absolute is not required here
Use display flex and justify the content to align in start and end